test/integration/deploy_defaults_test.go

```diff
@@ -75,7 +75,9 @@ func setEssentialDefaults(dc *deployapi.DeploymentConfig) *deployapi.DeploymentC
 		MaxSurge:            intstr.FromString("25%"),
 	}
 	dc.Spec.Strategy.ActiveDeadlineSeconds = int64ptr(21600)
-	dc.Spec.Triggers = []deployapi.DeploymentTriggerPolicy{}
+	dc.Spec.Triggers = []deployapi.DeploymentTriggerPolicy{
+		{Type: deployapi.DeploymentTriggerOnConfigChange},
```

Comment:
this reflects the reality... if you send an DC with `nil` triggers to server it will be defaulted to have config change trigger and send back...

@tnozicka @kargakis probably interesting for you.